tools/ci_build/build.py

```diff
@@ -989,13 +967,9 @@ def main():
                 cmake_extra_args = ['-A','x64','-T', toolset, '-G', args.cmake_generator]
             if args.enable_wcos:
                 cmake_extra_args.append('-DCMAKE_TOOLCHAIN_FILE=' + os.path.join(source_dir, 'cmake', 'wcos_toolchain.cmake'))
-        
-        if (args.android or args.ios) and not args.path_to_protoc_exe:
-            # Cross-compiling for Android and iOS
+        if args.android:
+            # Cross-compiling for Android
             path_to_protoc_exe = build_protoc_for_host(cmake_path, source_dir, build_dir, args)
-        else :
-            path_to_protoc_exe = args.path_to_protoc_exe
```

Comment:
This was the culprit for the pipeline break, it's overriding the one set when compiling protoc for the host (see `path_to_protoc_exe = build_protoc_for_host(cmake_path, source_dir, build_dir, args)`) some 50 lines above.